ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml

```diff
@@ -2291,6 +2291,415 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 0 0 4,11,18,25 * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+    timeout: 7h0m0s
+  extra_refs:
+  - base_ref: main
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale
+    ci-operator.openshift.io/variant: aws-4.17-nightly-x86
+    ci.openshift.io/generator: prowgen
+    job-release: "4.17"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.17-nightly-x86-control-plane-120nodes
```

Comment:
Please add slack reporter config in periodics file

Reply:
did someone take out all the reporter configs? I dont see any anymore in this file

Reply:
I find them removed in commit ebe0c165c19491b12f1e3836209fd4c0e43564a3